theming-preview/button.html

```diff
@@ -15,11 +15,8 @@
         .fd-theming .fd-theming__margin-bottom-03{
             margin-bottom: .3rem;
         }
-        @font-face { font-family: "72"; src: url("/themes/Base/baseLib/baseTheme/fonts/72-Regular-full.woff") format("woff"); font-weight: normal; font-style: normal; }
-        @font-face { font-family: "72"; src: url("/themes/Base/baseLib/baseTheme/fonts/72-Bold-full.woff") format("woff"); font-weight: 700; font-style: normal; }
-        @font-face { font-family: "72"; src: url("/themes/Base/baseLib/baseTheme/fonts/72-Light-full.woff") format("woff"); font-weight: 300; font-style: normal; }
-        @font-face { font-family: "SAP-icons"; src: url("/themes/Base/baseLib/baseTheme/fonts/SAP-icons.woff") format("woff"); font-weight: normal; font-style: normal; }
     </style>
+    <script src="ThemeManager.js"></script>
```

Comment:
not a big deal but we usually name our js files using kebab case aka`theme-manager.js`. Camel case is for the class names